CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # heroku-buildpack-php CHANGELOG
 
+## v141 (2018-08-07)
+
+### ADD
+
+- ext-redis/4.1.1 [David Zuelke]
+- ext-mongodb/1.5.2 [David Zuelke]
+
+### CHG
+
+- Verbose error messasge on `bin/detect` failure [David Zuelke]
+- Emit brief warnings for common regexed build failure cases [David Zuelke]
+- Run most internal 'composer' invocations using '--no-plugins' [David Zuelke]
```

Comment:
Why? Can you link this to an issue?

Reply:
There is an issue link in the commit message and above the first line that uses `--no-plugins` :)